### ISSUE TITLE
Remove deprecated warning when doing pnpm install

### ISSUE THREAD

**Issue by skull8888888** [lmnr-ai/lmnr#49]
Basically solve this
![Screenshot 2024-10-15 at 7 53 00 PM](https://github.com/user-attachments/assets/d53515ee-9f68-4f72-b990-8273bddeda76)

**Comment by Banh983** [comment]
Hi there! i would love to see if i can contribute anything for solving your problem, can you assgin it to me.

**Comment by Banh983** [comment]
I think that some of your package needed update to work. And the config-array@0.11.14, like in your image suggest you should install the @eslint/config-array instead. If those step not work feel free to ask

**Comment by dinmukhamedm** [comment]
Right, I had a quick look, we depend on most deprecated packages through `eslint@8`, which has been deprecated for a few months now. However, Next does not work with `eslint@9`, see https://github.com/vercel/next.js/issues/64409#issuecomment-2411304733. In [#55] I am removing one more dependency (`react-query`, now part of `@tanstack` and apparently unused) that brought some of the deprecated ones transitively.

But other than that, we'll just have to wait for Next 15 release.

**Comment by dinmukhamedm** [comment]
^ Now that React 19 is finally stable, I will attempt the Next 15 upgrade within the following couple weeks (assuming all conflicting dependencies such as some of shadcn and recharts quickly upgrade).

Good side effect is that `instrumentation.ts` is also stable from Next 15.

**Comment by dinmukhamedm** [comment]
The latest warning is 
```
2 deprecated subdependencies found: @esbuild-kit/core-utils@3.3.2, @esbuild-kit/esm-loader@2.6.5
```

and is coming from drizzle-kit. Options are either remove drizzle-kit from dev deps (and make sure everyone installs drizzle kit globally) or open an issue to them so they migrate to tsx (unlikely to be quick).

Closing this as the main issue has been solved
